sdk/keyvault/Azure.Security.KeyVault.Administration/tests/samples/Sample2_SelectiveRestore.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using System;
+
+namespace Azure.Security.KeyVault.Administration.Tests
+{
+    public class Sample2_SelectiveRestore : BackupRestoreTestBase
+    {
+        public Sample2_SelectiveRestore(bool isAsync) : base(isAsync, RecordedTestMode.Playback /* To record tests, change this argument to RecordedTestMode.Record */)
+        { }
+
+        [RecordedTest]
+        public async Task BackupAndRestoreSampleAsync()
+        {
+            var blobStorageUrl = TestEnvironment.StorageUri;
+            var blobContainerName = BlobContainerName;
+            var sasToken = "?" + SasToken;
+
+            // Create a Uri with the storage container.
+            UriBuilder builder = new UriBuilder(blobStorageUrl)
+            {
+                Path = blobContainerName,
+            };
+
+            // Start the backup.
+            BackupOperation backupOperation = await Client.StartBackupAsync(builder.Uri, sasToken);
+
+            // Wait for completion of the BackupOperation.
+            Response<Uri> backupResult = await backupOperation.WaitForCompletionAsync();
```

Comment:
Tip: with the implicit cast operator, you could just assign to `Uri backupResult` and elide the line below (line 36). We tend to do this more often in examples because it's cleaner.

The *only* problem is if customers use `var` (I tend to, though I don't with our ret vals for Azure SDK because of the implicit operators) they need to do `.Value`, so at least this establishes a precedent.

Reply:
I believe @tg-msft found that user studies found that not using the implicit cast in examples lead to greater success.